### DISCUSSION TITLE
Unable to show disabled cucumber scenarios in Allure report as Skipped

### DISCUSSION THREAD

**Discussion by saihaemmadi** [allure-framework/allure2#1973] [Questions & Support]
TestFramework: Cucumber with JUnit5
Issue: Disabled scenarios are not shown in allure report
Expected: Skipped/Disabled scenarios should be listed under skipped status in Allure report

Thanks in advance for looking into it.

**Comment by Watcher24** [comment]
We "struggled" with the same thing - and we came up with this solution (in Java):

In one of the classes scanned by cucumber for Hooks, add such method:
```
@Before (value = "@disabled",order=0)
    public void skipDisabledScenario(Scenario scenario) {
        // This Assumption will mark the scenario/testcase as Skipped in allure-report and scenario will not be executed as well
        assumeTrue(false, "This scenario is disabled and hence skipped");
    }
```

In the feature file, apply the tag `@disabled` accordingly. 
The cucumber-tag name is arbitrary
